packages/remediations/src/steps/ExistingOrNew.js

```diff
@@ -12,95 +11,136 @@ import {
-                <Stack gutter='md'>
-                    <StackItem>
-                        <h1 className='ins-m-text__bold'>Do you want to modify an existing Playbook or create a new one?</h1>
-                    </StackItem>
-                    <StackItem>
-                        <Grid>
-                            <GridItem sm={ 12 } md={ 6 } lg={ 3 }>
-                                <Radio
-                                    label={ existingRemediations ? `Existing Playbook (${existingRemediations.length})` : 'Existing Playbook' }
-                                    aria-label="Existing Playbook"
-                                    id="existing"
-                                    name="radio"
-                                    isDisabled={ !existingRemediations || !existingRemediations.length }
-                                    defaultChecked={ !props.state.isNewSwitch }
-                                    onChange={ () => props.onIsNewSwitch(false) }
-                                />
-                            </GridItem>
-                            <GridItem sm={ 12 } md={ 6 } lg={ 4 }>
-                                {
-                                    existingRemediations === false ?
-                                        <Skeleton size={ SkeletonSize.lg } /> :
-                                        <FormSelect
-                                            isDisabled={ isNewSwitch }
-                                            onChange={ props.onRemediationSelected }
-                                            value={ selectedRemediationId }
-                                            aria-label="Select an existing Playbook" >
-                                            { existingRemediations.length
-                                                ? existingRemediations.map(({ id, name }) =>
-                                                    <FormSelectOption key={ id } value={ id } label={ name } />)
-                                                :   <FormSelectOption key="empty" value="empty" label="No exising Playbooks" />
-                                            }
-                                        </FormSelect>
+const ExistingOrNewStep = ({
+    onIsNewSwitch,
+    onRemediationSelected,
+    onNameChange,
+    name,
+    nameValid,
+    isNewSwitch,
+    existingRemediations,
+    selectedRemediationId,
+    open: { data }
+}) => (<Form className="ins-c-existing-or-new">
+    <Stack gutter='md'>
+        <StackItem>
+            <span>
+                    You have selected&nbsp;
+                <h1 className='ins-m-text__bold'>
+                    {data.systems.length} {pluralazie('system', data.systems.length)}
+                </h1>
+                    &nbsp;with&nbsp;
+                <h1 className='ins-m-text__bold'>
+                    {data.issues.length} {pluralazie('issue', data.issues.length)}
+                </h1>
+                    &nbsp;to add to a remedation plan.
```

Comment:
The mocks modify terminology but that is not in effect currently. The old terminology should be used as described in https://marvelapp.com/j3b9b7e/screen/63083483
I.e., should say "Playbook" instead of "remediation plan"
Sorry for confusion.